tests/test_diffraction_objects.py

```diff
@@ -164,11 +164,11 @@ def test_diffraction_objects_equality(inputs1, inputs2, expected):
 
 
 def test_on_xtype():
-    test = DiffractionObject(wavelength=2 * np.pi, xarray=np.array([30, 60]), yarray=np.array([1, 2]), xtype="tth")
-    assert np.allclose(test.on_xtype("tth"), [np.array([30, 60]), np.array([1, 2])])
```

Comment:
using the instance `do` seems like a better wording here?

Reply:
Yes, test is now such a good choice.